README.md

```diff
@@ -1 +1,46 @@
 # osprey-flask-app
+
+This application is a flask microservice to interact with PCIC's [osprey](https://github.com/pacificclimate/osprey#readme) bird, which runs the RVIC streamflow package as a WPS process.
+
+## Installation
+We use `make` to handle the installation process. Copy and paste this section into your terminal:
+```
+make
+source /tmp/osprey-flask-app-venv/bin/activate
+```
+
+## Run App
+
+Before the `Flask` app can be started, an environment variable must be initialized using the following command:
+```
+export FLASK_APP=wsgi.py
+```
+
+Optionally, the app can be run in `development` mode using
+```
+export FLASK_ENV=development
+```
+
+After initializing these variables, the app can be started by running the following command (note that `host` and `port` are optional arguments)
+```
+flask run --host=<host> --port=<port>
+```
```

Comment:
This is something we could add to the `Makefile` so the user can just type `make run` and all the env vars would be handled.